cmd/detectExecuteScan.go

```diff
@@ -291,22 +290,7 @@ func getDetectScript(config detectExecuteScanOptions, utils detectUtils) error {
 		log.Entry().Infof("Using Detect Rescan script")
 		return utils.DownloadFile("https://raw.githubusercontent.com/blackducksoftware/detect_rescan/master/detect_rescan.sh", "detect.sh", nil, nil)
 	}
-	env := utils.GetOsEnv()
```

Comment:
There is already a detect8 version out. Maybe we can consider to recycle the code logic once we also upgrade to detect8 and need backwards compatibility to detect7. @andrew-kireev What do you think?

Reply:
We would like to test changes in detect version 8 and only after it we will add the same logic for downloading 8 version as default and version 7 if user specified it in env variable. I prefer to merge this pr to fix current issue for few users if you don't mind and return this logic after.